`typedef int Tcl_Size` in src/tclsqlite.c conflicts with `-DTcl_Size=int`.

--- a/autoconf/tea/tclconfig/tcl.m4
+++ b/autoconf/tea/tclconfig/tcl.m4
@@ -3139,10 +3139,6 @@
 	fi
     fi
 
-    if test "${TCL_MAJOR_VERSION}" -lt 9 -a "${TCL_MINOR_VERSION}" -lt 7; then
-	    AC_DEFINE(Tcl_Size, int, [Is 'Tcl_Size' in <tcl.h>?])
-	fi
-
     #--------------------------------------------------------------------
     # Common compiler flag setup
     #--------------------------------------------------------------------
